content/video-production/homepage-videos.mdx

```diff
@@ -0,0 +1,46 @@
+---
+seo:
+  title: Homepage Videos
+  description: ''
```

Comment:
bad SEO - copy meta desc from old page or make one up or ask @camillars

Reply:
Hi @jackreimers 

the SEO implemented does not describe this page. :(

1. please use the below
title - Video Production for Business & Product Success
description - Make your brand stand out with captivating product videos that impress your target audience and leave a lasting impact. (119 characters)